go.mod

```diff
@@ -4,39 +4,52 @@ go 1.13
 
 require (
 	cloud.google.com/go/storage v1.11.0 // indirect
+	github.com/Azure/azure-sdk-for-go v50.2.0+incompatible // indirect
+	github.com/Azure/go-autorest/autorest v0.11.17 // indirect
+	github.com/Azure/go-autorest/autorest/adal v0.9.10 // indirect
+	github.com/Azure/go-autorest/autorest/to v0.4.0 // indirect
+	github.com/Azure/go-autorest/autorest/validation v0.3.1 // indirect
 	github.com/GoogleCloudPlatform/cloud-builders/gcs-fetcher v0.0.0-20191203181535-308b93ad1f39
+	github.com/aws/aws-sdk-go v1.37.1 // indirect
 	github.com/cloudevents/sdk-go/v2 v2.1.0
+	github.com/docker/cli v20.10.2+incompatible // indirect
+	github.com/docker/docker v20.10.2+incompatible // indirect
```

Comment:
That's a lot of new deps 😅

Reply:
the deps have been there before - unclear when the go toolchain adds it to the `go.mod` file with `//indirect`